lib/fortuneteller/job/day_plan.rb

```diff
@@ -4,12 +4,15 @@ class DayPlan < FortuneTeller::Base::DayPlan
       attr_accessor :base, :savings_plans
       def initialize(*args)
         @savings_plans = []
-        @adjusted_base = {}
         super
       end
 
       def set(base:)
-        @base = base
+        @base = FortuneTeller::Utils::InflatingInt.new(
```

Comment:
This ties back into the longer comment about future dollars vs todays dollars.  Can we refector to avoid initializing InflatingInt directly, and instead use `sim.inflating_int` so that this uses todays dollars?

Reply:
Sure, we can do that. I made the assumption here that any `set` call here would be setting the base salary exactly as of the given date, so we wouldn't want to immediately inflate it, but your comments below make sense.